test/index.test.js

```diff
@@ -492,11 +492,11 @@ test('AWS.setSDK function should mock a specific AWS module', function(t) {
   });
 
   t.test('Modules with multi-parameter constructors can be set for mocking', function(st) {
-	awsMock.setSDK('aws-sdk');
-	awsMock.mock('CloudFront.Signer', 'getSignedUrl');
-	var signer = new AWS.CloudFront.Signer('key-pair-id', 'private-key');
-	st.ok(signer);
-	st.end();
+    awsMock.setSDK('aws-sdk');
+    awsMock.mock('CloudFront.Signer', 'getSignedUrl');
+		var signer = new AWS.CloudFront.Signer('key-pair-id', 'private-key');
```

Comment:
should this `var` not bee in-line with the other lines?

Reply:
Good catch. VS Code gets confused because some of by projects use tabs with tabstop 2 and others use spaces. Corrected.